cypress/e2e/tests/migration/reports/appsrisk.test.ts

```diff
@@ -16,30 +16,25 @@ limitations under the License.
 /// <reference types="cypress" />
 
 import {
-    hasToBeSkipped,
     login,
     preservecookies,
     clickByText,
     createMultipleStakeholders,
     createMultipleApplications,
-    deleteAllStakeholders,
-    deleteApplicationTableRows,
     selectUserPerspective,
-    createMultipleBusinessServices,
+    deleteByList,
 } from "../../../../utils/utils";
 import { verifyApplicationRisk } from "../../../models/migration/reports/reports";
 import { Assessment } from "../../../models/migration/applicationinventory/assessment";
 import { navMenu } from "../../../views/menu.view";
-import { businessService, migration, reports } from "../../../types/constants";
+import { migration, reports, SEC } from "../../../types/constants";
 import { Stakeholders } from "../../../models/migration/controls/stakeholders";
-import { BusinessServices } from "../../../models/migration/controls/businessservices";
 
-var stakeholdersList: Array<Stakeholders> = [];
-var applicationsList: Array<Assessment> = [];
-var businessServiceList: Array<BusinessServices> = [];
+let stakeholdersList: Array<Stakeholders> = [];
+let applicationsList: Array<Assessment> = [];
 
 describe(["@tier2"], "Application risks tests", () => {
-    var risktype = ["low", "medium", "high"];
+    let riskType = ["low", "medium", "high"];
```

Comment:
**nitpick:** This array can be declared as a `const` as it isn't reassigned in the whole suite.

This comment applies to multiple variables updated in this PR.

[More info](https://eslint.org/docs/latest/rules/prefer-const)

Reply:
Actually I can move it to constants and import if from there, but that will be next time